biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -2468,25 +2618,52 @@ subroutine terminate_patches(currentSite)
                 ! it will be returned by the subroutine as de-referenced
                 
                 olderPatch => currentPatch%older
-                call fuse_2_patches(currentSite, olderPatch, currentPatch)
-                
-                ! The fusion process has updated the "older" pointer on currentPatch
-                ! for us.
+
+                if (currentPatch%anthro_disturbance_label .eq. olderPatch%anthro_disturbance_label) then
+                   
+                   call fuse_2_patches(currentSite, olderPatch, currentPatch)
                 
-                ! This logic checks to make sure that the younger patch is not the youngest
-                ! patch. As mentioned earlier, we try not to fuse it.
+                   ! The fusion process has updated the "older" pointer on currentPatch
+                   ! for us.
                 
-             elseif( associated(currentPatch%younger) ) then
+                   ! This logic checks to make sure that the younger patch is not the youngest
+                   ! patch. As mentioned earlier, we try not to fuse it.
+                   
+                   gotfused = .true.
+                else
+                   if (count_cycles .gt. 0) then
+                      ! if we're having an incredibly hard time fusing patches because of their differing anthropogenic disturbance labels, 
+                      ! since the size is so small, let's sweep the problem under the rug and change the tiny patch's label to that of its older sibling
```

Comment:
```suggestion
                      ! since the size is so small, let's sweep the problem under the rug and change the tiny patch's label to that of its older sibling
                      ! and then allow them to fuse together. 
```